src/components/TodoInput/TodoInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInput: React.FC<Props> = ({ todo }) => (
+  <div
+    data-cy="Todo"
+    className={classNames('todo', { completed: todo.completed })}
+  >
+    {/* eslint-disable-next-line jsx-a11y/label-has-associated-control*/}
```

Comment:
Don't disable linter

Reply:
Linter blocks deploying, cause he needs onChange event on this form, which is 100% no needed in current task. That's why i blocked him...